cmd/livepeer/starter/starter.go

```diff
@@ -1412,3 +1417,31 @@ func getBroadcasterPrices(broadcasterPrices string) []BroadcasterPrice {
 
 	return pricesSet.Prices
 }
+
+type KeystorePath struct {
+	path, address interface{}
```

Comment:
Can we use `string` here? Then there would be no need for casting.

```suggestion
	path, address string
```